tests/migrations/test_operations.py

```diff
@@ -52,8 +52,9 @@ def make_test_state(self, app_label, operation, **kwargs):
 
     def set_up_test_model(
             self, app_label, second_model=False, third_model=False, multicol_index=False,
-            related_model=False, mti_model=False, proxy_model=False, manager_model=False,
-            unique_together=False, options=False, db_table=None, index_together=False):
+            related_model=False, o2o_model=False, mti_model=False, proxy_model=False,
```

Comment:
I guess these tests would be better in `tests/schema` since this doesn't involve fixing operations at all. Did you have any thinking about that?